### PR TITLE
feat: add styling to page

### PR DESCRIPTION
**Pull request by Amrhub**
- add loader animation (note I set `setTimout` to stimulate real loading and give you a chance to see it) 
- add some styling to hovering/focus/active

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 # LeaderBoard
 
-> LeaderBoard functionality
```
Comment (omar-labana) [inline README.md:5]: - [x] README file should contain all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them (if necessary)

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @Amrhub, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Amrhub) [reply README.md:5]: Yes but that was not part of requirements I could just pul a link to their documentations, would it do ?

**Review by mikethreels — APPROVED** [review]
Hi Amr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**